source/loader/layers/sanitizer/asan_libdevice.hpp

```diff
@@ -62,6 +62,23 @@ struct DeviceSanitizerReport {
     bool IsRecover = false;
 };
 
+struct LocalArgsInfo {
+    uint64_t Size = 0;
+    uint64_t SizeWithRedZone = 0;
+};
+
+struct LaunchInfo {
+    uintptr_t PrivateShadowOffset =
+        0; // don't move this field, we use it in AddressSanitizerPass
```

Comment:
to help me better understand the above comment, can we move other fields?

Reply:
Yes, you can reorder other fields except `PrivateShadowOffset`, because we assume `PrivateShadowOffset` is the first field in ASanPass.